oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/Column.java

```diff
@@ -58,6 +58,12 @@
      */
     boolean storageOnly() default false;
 
+    /**
+     * The column(field) is just indexed, never stored. Note: this feature only supported by elasticsearch
+     * and can not support update mappings in current index
```

Comment:
```suggestion
     * and don't support mappings update due to ElasticSearch server's limitation.
     *
     * NOTICE, metrics should not use this, as the OAP core merges indices of metrics automatically.
```

`can't` usually means we can not originally.